src/masking.jl

```diff
@@ -28,7 +28,7 @@ make_mask(::FlatField{<:Flat{Nside,θpix}}; kwargs...) where {Nside,θpix} = mak
 # all padding/smoothing/etc... quantities below are in units of numbers of pixels
 
 function boundarymask(Nside, pad)
-    m = fill(true,Nside,Nside)
+    m = typeof(Nside)<:Tuple ? fill(true,Nside[2],Nside[1]) : fill(true, Nside, Nside)
```

Comment:
And here maybe `fill(true, (Nside .* (1,1))...)`

(Sorry this is so minor, just trying to be more concise and get rid of the tiny code duplication. If you do want to keep these anywhere, note `Nside isa Tuple` is equivalent to `typeof(Nside)<:Tuple` and slighly more common)

Reply:
No, these are all good suggestions!